.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please follow the template so that the reviewers can easily understand what the code changes affect -->
+
+# Summary
+
+<!-- Explain the **motivation** for making this change. What issues does the pull request solve? Please tag them so that they will get automatically closed once the PR is merged -->
+
+## Test Plan
```

Comment:
What do you think of we add the following topics?

After `Summary` as a subtopic under it:
 - What's the feature?
 - What's the solution?
 - What areas of the application does it impact?

We could also add two sections here under `Test Plan`:
 - What's required for testing (prerequisites)?
 - What are the steps to reproduce (after prerequisites)?
 - Other Notes

Do you like the proposed ideas? :)

Reply:
I think that we need to find a balance of how many different things we want them to write, I'll modify the template to reflect your feedback + Mike's